### PR TITLE
FD production changes

### PR DESCRIPTION
**Pull request by absolution1**
Can now be merged.

 - Make the refactored workflow the standard workflow for the FD1-HD
 - Use wirecell as the standard workflow for FD1-HD
 - Combined sigsim+sigproc wirecell is the standard for FD1-HD
 - Provided separated sigsim and sigproc workflows for FD1-HD
 - reco1 and reco2 workflows for both FD1 and FD2
 -  hitfd as input to pandora rather than line cluster (decouple unsupported clustering from supported clustering)
 - remove energyrecpmtracktc from VD (not needed)

### CODE REVIEW

**Comment by tomjunk** [comment]
This PR still says please do not merge yet.  Still the case?

**Comment by absolution1** [comment]
Hi @tomjunk 
This PR is good to go now.

File: fcl/dunefdvd/reco/workflow_reco_dunevd10kt.fcl [hunk 1]
```diff
@@ -66,29 +90,57 @@ dunefd_vertdrift_tpc:
     cvneval,
     energyrecnumu,
     energyrecnue,
-    energyrecnc
+    energyrecnue
```
Comment (lpaulucc) [inline fcl/dunefdvd/reco/workflow_reco_dunevd10kt.fcl:93]: energyrecnue twice?

**Review by lpaulucc — COMMENTED** [review]
Tiny comment in one of the fhicls.
The other 49 files look ok to me.

File: fcl/dunefdvd/reco/workflow_reco_dunevd10kt.fcl (hunk 1 above)
Comment (absolution1) [reply fcl/dunefdvd/reco/workflow_reco_dunevd10kt.fcl]: Good catch!
Corrected to be energyrecnc

**Review by lpaulucc — APPROVED** [review]

**Pull request merged**